nebula_pytest_bdd/conftest.py

```diff
@@ -0,0 +1,202 @@
+import time
+import sys
+import pytest_bdd
+import pytest
+import json
+import os
+sys.path.insert(0,'./nebula-clients/python/')
+from nebula2.common import ttypes
+from nebula2.gclient.net import ConnectionPool
+from nebula2.Config import Config
+
+
+
+
+@pytest.fixture(scope='session')
+def init_client():
+    conf = Config()
+    jsonfile = "nebula.json"
+    data = json.load(open(jsonfile))
+    conf.timeout = data['timeout']
+    conf.idle_time = data['idle_time']
+    conf.max_connection_pool_size = data['max_connection_pool_size']
+    conf.max_retry_time = data['max_retry_time']
+    addresses = list()
+    addresses.append((data['ip'],data['port']))
+    connection_pool = ConnectionPool()
+    connection_pool.init(addresses,conf)
+    client_list = list()
+    client_list.append(connection_pool)
+    client_list.append(data['user'])
```

Comment:
the `username` and `password` can use differently, so here just return the pool it ok.